src/numberparse.rs

```diff
@@ -133,26 +129,14 @@ pub enum Number {
 }
 
 #[cfg_attr(not(feature = "no-inline"), inline)]
+#[cfg(target_arch = "aarch64")]
```

Comment:
We’d probably want to move number parse in a x86 module and create a corresponding arm one instead of changing it to non simd code to cover both cases